### PR TITLE
kafka: add support for disk usage metrics

### PR DESCRIPTION
**Pull request by shubhanilBag**
This PR -
1. Adds a [prometheus node exporter](https://github.com/prometheus/node_exporter) sidecar
    container which helps in exposing properly tagged disk usage metrics for PV attached to the
   kudo-kafka instance
2. Updates grafana dashboard to show the disk usage metrics
grafana-capture
Screenshot from 2019-11-26 22-21-14
3. Updates monitoring doc

### CODE REVIEW

File: repository/kafka/operator/templates/statefulset.yaml [hunk 1]
```diff
@@ -23,6 +23,29 @@ spec:
     spec:
       terminationGracePeriodSeconds: 300
       containers:
+        {{ if eq .Params.METRICS_ENABLED "true" }}
+        - name: kafka-node-exporter
+          image: quay.io/prometheus/node-exporter
+          resources:
+            requests:
+              memory: 100Mi
```
Comment (zmalik) [inline repository/kafka/operator/templates/statefulset.yaml]:
```suggestion
              memory: 20Mi
```

File: repository/kafka/operator/templates/statefulset.yaml [hunk 2]
```diff
@@ -23,6 +23,29 @@ spec:
...
+              cpu: 0.1
+            limits:
+              memory: 150Mi
```
Comment (zmalik) [inline repository/kafka/operator/templates/statefulset.yaml]:
```suggestion
              memory: 50Mi
```

**Review by zmalik — CHANGES_REQUESTED** [review]
thanks, @shubhanilBag LGTM!

just left a comment around memory request and limit around the sidecar, as I remember you monitored the memory usage and we agreed that request can be 20Mi and limit can 50Mi right?

**Comment by shubhanilBag** [comment]
> thanks, @shubhanilBag LGTM!
> 
> just left a comment around memory request and limit around the sidecar, as I remember you monitored the memory usage and we agreed that request can be 20Mi and limit can 50Mi right?

Yes! I updated the PR with the appropriate limits

**Review by zmalik — APPROVED** [review]
LGTM

**Pull request merged**